madgiraffestudios/src/components/About/About.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import photoshoots from '../../Images/aboutPage/photoshoot/brandon_shoot.jpg';
 import graphicdesign from '../../Images/aboutPage/graphics/graphic02.png';
 import drawing from '../../Images/aboutPage/drawings/lucy_ill.jpeg';
-import { SocialIcon } from 'react-social-icons';
```

Comment:
This was taken out during another PR

Reply:
No problem, means there will be no merge/rebase conflicts then